render-app/src/main/java/org/janelia/alignment/spec/TileBoundsRTree.java

```diff
@@ -304,21 +302,16 @@ public static Set<OrderedCanvasIdPair> getDistinctPairs(final TileBounds fromTil
                         isNeighborCenterInRange(fromMinX, fromMaxX, toTile.getMinX(), toTile.getMaxX()) ||
                         isNeighborCenterInRange(fromMinY, fromMaxY, toTile.getMinY(), toTile.getMaxY())) {
 
+                        final OrderedCanvasIdPair pair;
                         if (includeRelativePosition) {
-                            relativePositions = MontageRelativePosition.getRelativePositions(fromTile, toTile);
-                            pairs.add(
-                                    new OrderedCanvasIdPair(
-                                            new CanvasId(fromTile.getSectionId(), pTileId, relativePositions[0]),
-                                            new CanvasId(toTile.getSectionId(), qTileId, relativePositions[1]),
-                                            null));
+                            pair = OrderedCanvasIdPair.withRelativePositions(fromTile, toTile);
```

Comment:
Also here: if I set the `includeRelativePosition` flag, I would expect that the relative z-distance is included.

Reply:
makes sense - renamed this to make it clearedr